clarity/src/vm/docs/mod.rs

```diff
@@ -1978,8 +1979,9 @@ This function returns (ok true) if the transfer is successful, or, on an error,
 const STX_BURN: SimpleFunctionAPI = SimpleFunctionAPI {
     name: None,
     signature: "(stx-burn? amount sender)",
-    description: "`stx-burn?` decreases the `sender` principal's STX holdings by `amount`, destroying
-the STX. The `sender` principal _must_ be equal to the current context's `tx-sender`.
+    description: "`stx-burn?` decreases the `sender` principal's STX holdings by `amount`,
+specified in microstacks, destroying the STX. The `sender` principal _must_ be equal to the current
```

Comment:
`destroying the STX` --> `by destroying them`

Reply:
Addressed in 2bb0ee182.